.github/workflows/cypress-test.yml

```diff
@@ -9,95 +9,45 @@ env:
           EOF
-          docker build -t opensearch-test:latest -f- . <<EOF
-          FROM ubuntu:latest
-          COPY --chown=1001:1001 os-ep.sh /docker-host/
-          COPY --chown=1001:1001 opensearch-security.zip /docker-host/security-plugin.zip
-          COPY --chown=1001:1001 opensearch* /opensearch/
-          RUN chmod +x /docker-host/os-ep.sh
-          RUN useradd -u 1001 -s /sbin/nologin opensearch
-          ENV PATH="/opensearch/bin:${PATH}"
-          WORKDIR /opensearch/
-          ENTRYPOINT /docker-host/os-ep.sh
-          EOF
-          docker run -d -p 9200:9200 -p 9600:9600 -i opensearch-test:latest
 
-      - name: Checkout OpenSearch Dashboard
-        uses: actions/checkout@v2
-        with:
-          path: OpenSearch-Dashboards
-          repository: opensearch-project/OpenSearch-Dashboards
-          ref: 'main'
-          fetch-depth: 0
-      
-      - name: Create plugins dir
-        run: |
-          cd ./OpenSearch-Dashboards
-          mkdir -p plugins
-      
-      - name: Checkout OpenSearch Dashboard Security plugin
-        uses: actions/checkout@v2
+      - name: Run Opensearch with A Single Plugin
+        uses: opensearch-project/security/.github/actions/start-opensearch-with-one-plugin@main
         with:
-          path: OpenSearch-Dashboards/plugins/security-dashboards-plugin
-          ref: ${{ github.ref }}
-
-      - name: Check OpenSearch Running
-        continue-on-error: true
-        run: curl -XGET https://localhost:9200 -u 'admin:admin' -k
+          opensearch-version: ${{ env.OPENSEARCH_VERSION }}
+          plugin-name: ${{ env.PLUGIN_NAME }}
+          setup-script-name: setup
 
-      - name: Get node and yarn versions
-        id: versions
-        run: |
-          echo "::set-output name=node_version::$(cat ./OpenSearch-Dashboards/.node-version)"
-          echo "::set-output name=yarn_version::$(jq -r '.engines.yarn' ./OpenSearch-Dashboards/package.json)"
-      
-      - name: Setup node
-        uses: actions/setup-node@v1
+      - name: Run Dashboard with Security Dashboards Plugin
+        uses: ./.github/actions/install-dashboards
         with:
-          node-version: ${{ steps.versions.outputs.node_version }}
-          registry-url: 'https://registry.npmjs.org'
-      
-      - name: Install correct yarn version for OpenSearch Dashboards
-        run: |
-          npm uninstall -g yarn
-          echo "Installing yarn ${{ steps.versions_step.outputs.yarn_version }}"
-          npm i -g yarn@${{ steps.versions.outputs.yarn_version }}
-      
-      - name: Check OpenSearch Running
-        continue-on-error: true
-        run: curl -XGET https://localhost:9200 -u 'admin:admin' -k
+          plugin_name: security-dashboards-plugin
 
-      - name: Bootstrap OpenSearch Dashboards
+      - name: Configure and Run OpenSearch Dashboards
         continue-on-error: false
         run: |
           cd ./OpenSearch-Dashboards
-          yarn osd bootstrap
```

Comment:
are we completely sure that this step would run bootstrapping OSD?

Reply:
Ya we have already done the bootstrap inside `./github/actions/install-dashboards/action.yml`, so that we don't need to bootstrap again even with the configuration steps of `opensearch_dashboard.yml`.